easybuild/easyconfigs/s/STACEY/STACEY-1.2.5-GCC-10.2.0.eb

```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = 'Tarball'
+
+name = 'STACEY'
+version = '1.2.5'
+
+homepage = 'http://www.indriid.com/software.html'
+description = """The BEAST2 package STACEY can be used for species delimitation and species tree estimation,
+based on the multispecies coalescent model."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['http://www.indriid.com/2019/']
+sources = ['%(name)s.addon.v%(version)s.zip']
+checksums = ['a6ce3ce83ddafe3505622961c7d90e696e11b2b51246f7447d820e1b3e717b58']
+
+builddependencies = [
+    ('Xvfb', '1.20.9'),
+]
+dependencies = [
+    ('Beast', '2.6.4'),
+]
+
+# we need to link installdir to hardcoded package directory of Beast (~/.beast/2.6/)
+postinstallcmds = [
+    # if package dir doesnt exist yet, initial run of `beauti` creates it
+    'timeout 10 xvfb-run beauti 2>&1 | grep "Terminated"',
+    # now we can link our installdir to the package dir
+    'ln -s %(installdir)s ~/.beast/2.6/STACEY ',
```

Comment:
@deniskristak This won't work, since this symlink will only be put in place in the home directory of the user that is doing the installation...

Is there any way to instruct BEAST to take other directories next to `~/.beast/<version>` into account, maybe through some environment variable?

Reply:
I asked about this on the `beast-users` mailing list (see https://groups.google.com/g/beast-users/), but it seems like my posted didn't go through (or it's being moderated).

Reply:
@deniskristak Ah, there is some useful information in https://groups.google.com/g/beast-users/c/eU-J4byVP7Q/m/62bQhYIRAQAJ:

```
If you want to pick up packages in a non-standard directory, you need to start java with the “beast.user.package.dir” variable set, for example like so:

java -Dbeast.user.package.dir=/my/package/dir beast.app.beastapp.BeastLauncher beast.xml
```

There's some info in http://www.beast2.org/managing-packages/ too.

I think we can use a custom value for `beast.user.package.dir` by setting `$_JAVA_OPTIONS` when loading the `STACEY` module, which should be picked up automatically by the `java` command, so something like this:

```python
modextravars = {'_JAVA_OPTIONS': "-Dbeast.user.package.dir=%(installdir)s"}
```

in the STACEY easyconfig file. Can you try using that, rather than symlinking in `~/.beast`?

Reply:
this actually works, nice catch